R/data_fatima.R

```diff
@@ -0,0 +1,35 @@
+#' Do Americans Support Impeaching Trump?
+#' 
+#' Raw data behind this story
+#' "Do Americans Support Impeaching Trump?"
+#' \url{https://projects.fivethirtyeight.com/impeachment-polls/}
+#' 
+#' @format A data frame with 388 rows of polling data and 24 variables:
+#' \describe{
+#'   \item{start}{Poll start date, the first date responses were collected}
+#'   \item{end}{Poll end date, the last date responses were collected}
+#'   \item{pollster}{entity/organization that created poll, collected and published data}
+#'   \item{sponsor}{sponsor of pollster}
+#'   \item{sample_size}{number of respondents for each}
+#'   \item{pop}{categorical variable with 3 categories: a, rv, lv -- value unknown}
+#'   \item{tracking}{true/false logical -- value unknown}
+#'   \item{text}{poll question}
+#'   \item{category}{category of poll question with 5 categories: impeach and remove, begin proceedings, begin inquiry, reasons, impeach}
+#'   \item{include}{yes/no logical -- value unknown}
+#'   \item{yes}{% of repsondents in sample who answered "Yes" to the poll question}
```

Comment:
Unfortunately it seems all the following `%` signs are causing errors. Could you replace them all with the word `Percent`?

Reply:
Hi Albert! Sorry for the delay on this, I made the requested change above. Let me know if there are any other changes I need to make.

Reply:
Eep! It seems a merge conflict has creeped in as evidenced by the 
```
<<<<<<< HEAD
```
and the
```
=======
```
and the
```
>>>>>>> 25e8defc63e3a707dcecaa074932ec91ea22f474
```
Could you remove these lines, and remove the redundant entries? Almost there!

Reply:
I've removed the lines and redundant entries (hopefully), please let me know if you can see these changes on your end?